packages/services/f-feature-management/src/calculateFractions.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-bitwise */
+
+import CryptoJSMD5 from 'crypto-js/md5';
+import CryptoJS from 'crypto-js/core';
+
+// note: Crypto-JS works using word arrays, but the algorithm requires we have access to individual bytes
+//      There is probably a more efficient way to manipulate specific bytes without having to convert the whole
+//      word array.  These conversion functions were found on the internet.
+function byteArrayToWordArray (ba) {
+    const wa = [];
+    for (let i = 0; i < ba.length; i++) {
+        wa[(i / 4) | 0] |= ba[i] << (24 - 8 * i);
+    }
+
+    return CryptoJS.lib.WordArray.create(wa, ba.length);
+}
+
+function wordToByteArray (word, length) {
```

Comment:
Can you add JSDoc comments to any new functions? ([for example](https://github.com/justeat/fozzie-components/blob/master/packages/services/f-http/src/createClient.js) – doesn't need to be in this commit, but added as part of this work).

Looks like you've got them in some places below, but not everywhere.

Reply:
Done. I had previously only added to exported or interesting functions.